maths/mobius_function.py

```diff
@@ -0,0 +1,42 @@
+"""
+Refrences: https://en.wikipedia.org/wiki/M%C3%B6bius_function
+References: wikipedia:square free number
+python/black : True
+flake8 : True

```

Comment:
Did you run __flake8 maths/mobius_function.py__ on your local machine and did it pass?  If not then please change this line to __flake8 : --select=E9,F63,F7,F82__ to match https://github.com/TheAlgorithms/Python/blob/master/.travis.yml#L9

Reply:
$ flake8 .
```
./maths/mobius_function.py:8:1: F401 'typing.List' imported but unused
./maths/mobius_function.py:12:1: E302 expected 2 blank lines, found 1
./maths/mobius_function.py:39:1: E305 expected 2 blank lines after class or function definition, found 1
```
__python/black__ will autofix two of these.

Reply:
I fixed it. I ran flake 8 on the original code.